modules/nw-egress-router-cr.adoc

```diff
@@ -0,0 +1,118 @@
+:redirect:
+:router-type: redirect
+endif::[]
+:router-name: egress-router-{router-type}
+
+[id="nw-egress-router-ovn-cr_{context}"]
+= Egress router custom resource
+
+Define the configuration for an egress router pod in an egress router custom resource. The following YAML describes the fields for the configuration of an egress router in {router-type} mode:
+
+// cluster-network-operator/manifests/0000_70_cluster-network-operator_01_egr_crd.yaml
+[source,yaml,subs="attributes+"]
+----
+apiVersion: network.operator.openshift.io/v1
+kind: EgressRouter
+metadata:
+  name: <egress_router_name>
+spec:
+  addresses: [  <.>
+    {
+      ip: "<egress_router>",  <.>
+      gateway: "<egress_gateway>"  <.>
+    }
+  ]
+  networkInterface: {  <.>
+    macvlan: {
+      master: "<master_interface>",  <.>
+      mode: "<macvlan_mode>"  <.>
+    }
+  }
+  mode: Redirect
+  redirect: {
+    redirectRules: [  <.>
+      {
+        destinationIP: "<egress_destination>",
+        port: <egress_router_port>,
+        targetPort: <target_port>,  <.>
+        protocol: <network_protocol>  <.>
+      },
+      ...
+    ]
+    fallbackIP: "<egress_destination>"  <.>
+  }
+----
+<.> The `addresses` field specifies the IP addresses to configure on the secondary network interface.
+
+<.> The `ip` field specifies the reserved source IP address from the physical network that the node is on to use with egress router pod.
+
+<.> The `gateway` field specifies the IP address of the network gateway.
+
+<.> The `networkInterface` field specifies the network interface for the egress router to use. The supported interface type is macvlan.
+
+<.> Optional: The `master` field specifies the name of the master network interface. You do not need to specify this field if the interface can be inferred from the IP address that is specified in the `addresses.ip` field.
+
+<.> The `mode` field specifies the mode for the macvlan network interface. Specify one of Bridge, Private, VEPA, or Passthru. The default value is "Bridge".
+
+<.> Optional: The `redirectRules` field specifies a combination of egress destination IP address, egress router port, and protocol. Incoming connections to the egress router on the specified port and protocol are routed to the destination IP address.
+
+<.> Optional: The `targetPort` field specifies the network port on the destination IP address. If this field is not specified, then traffic is routed to the same network port that it arrived on.
+
+<.> The `protocol` field supports TCP, UDP, or SCTP.
+
+<.> Optional: The `fallbackIP` field specifies one IP address for a remote destination. If redirect rules are not specified, all traffic from the router is redirected to this destination. If redirect rules are specified, any connections to the egress router on any port that is not defined in a rule is redirected to this destination. If redirect rules are specified and no fallback IP address is provided, connections to the egress router on other ports are rejected.
```

Comment:
I'd remove fallbackIP for now, this was added to the CRD, but won't be funcional for now.